Tests/TestingApps/PaywallsTester/PaywallsTester/Views/OfferingsList.swift

```diff
@@ -41,23 +41,13 @@ struct OfferingsList: View {
         switch self.offerings {
         case let .success(offerings):
             self.list(with: offerings)
+            #if !targetEnvironment(macCatalyst)
```

Comment:
Hmm I wonder if we should provide the option of presenting as a sheet somewhere... But I guess we can do that later. This does provide more flexibility 👍